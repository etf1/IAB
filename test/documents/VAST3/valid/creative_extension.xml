<!-- <Creative>s with different <CreativeExtensions> values -->
<VAST version="3.0">
    <Ad id="601364">
        <InLine>
            <AdSystem>Acudeo Compatible</AdSystem>
            <AdTitle>VAST 2.0 Instream Test 1</AdTitle>
            <Description>VAST 2.0 Instream Test 1</Description>
            <Error>http://myErrorURL/error</Error>

            <Impression>http://myTrackingURL/impression</Impression>
            <Creatives>
                <Creative AdID="601364">
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <CreativeExtensions></CreativeExtensions>
                    </Linear>
                </Creative>
                <Creative AdID="601365">
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <CreativeExtensions>
                            <CreativeExtension></CreativeExtension>
                        </CreativeExtensions>
                    </Linear>
                </Creative>
                <Creative AdID="601366">
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <CreativeExtensions>
                            <CreativeExtension>
                                <haptic creativeId="12727552"><![CDATA[http://slpubmedias.tf1.fr/MAX/HAPTIC/CARTENOIRE/V3/CarteNoir.hapt]]></haptic>
                            </CreativeExtension>
                        </CreativeExtensions>
                    </Linear>
                </Creative>
                <Creative AdID="601367">
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <CreativeExtensions>
                            <CreativeExtension>
                                <haptic creativeId="12727552"><![CDATA[http://slpubmedias.tf1.fr/MAX/HAPTIC/CARTENOIRE/V3/CarteNoir.hapt]]></haptic>
                                <haptic creativeId="12727552"><![CDATA[http://slpubmedias.tf1.fr/MAX/HAPTIC/CARTENOIRE/V3/CarteNoir.hapt]]></haptic>
                            </CreativeExtension>
                        </CreativeExtensions>
                    </Linear>
                </Creative>
                <Creative AdID="601368">
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <CreativeExtensions>
                            <CreativeExtension>
                                <haptic creativeId="12727552"><![CDATA[http://slpubmedias.tf1.fr/MAX/HAPTIC/CARTENOIRE/V3/CarteNoir.hapt]]></haptic>
                            </CreativeExtension>
                            <CreativeExtension>
                                <haptic creativeId="12727552"><![CDATA[http://slpubmedias.tf1.fr/MAX/HAPTIC/CARTENOIRE/V3/CarteNoir.hapt]]></haptic>
                            </CreativeExtension>
                        </CreativeExtensions>
                    </Linear>
                </Creative>
            </Creatives>
        </InLine>
    </Ad>
</VAST>
