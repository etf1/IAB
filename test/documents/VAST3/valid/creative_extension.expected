{
  "version": "3.0",
  "ads": [
    {
      "adType": 1,
      "adTitle": "VAST 2.0 Instream Test 1",
      "description": "VAST 2.0 Instream Test 1",
      "adSystem": {
        "name": "Acudeo Compatible"
      },
      "id": "601364",
      "error": "http://myErrorURL/error",
      "impressions": [
        {
          "uri": "http://myTrackingURL/impression"
        }
      ],
      "creatives": [
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "adID": "601364"
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "adID": "601365",
          "extensions": [
            {}
          ]
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "adID": "601366",
          "extensions": [
            {
              "haptic": [
                {
                  "creativeId": "12727552",
                  "$t": "http://slpubmedias.tf1.fr/MAX/HAPTIC/CARTENOIRE/V3/CarteNoir.hapt"
                }
              ]
            }
          ]
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "adID": "601367",
          "extensions": [
            {
              "haptic": [
                {
                  "creativeId": "12727552",
                  "$t": "http://slpubmedias.tf1.fr/MAX/HAPTIC/CARTENOIRE/V3/CarteNoir.hapt"
                },
                {
                  "creativeId": "12727552",
                  "$t": "http://slpubmedias.tf1.fr/MAX/HAPTIC/CARTENOIRE/V3/CarteNoir.hapt"
                }
              ]
            }
          ]
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "adID": "601368",
          "extensions": [
            {
              "haptic": [
                {
                  "creativeId": "12727552",
                  "$t": "http://slpubmedias.tf1.fr/MAX/HAPTIC/CARTENOIRE/V3/CarteNoir.hapt"
                }
              ]
            },
            {
              "haptic": [
                {
                  "creativeId": "12727552",
                  "$t": "http://slpubmedias.tf1.fr/MAX/HAPTIC/CARTENOIRE/V3/CarteNoir.hapt"
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}
